README.md

```diff
@@ -59,7 +59,7 @@ To create a JuiceFS, you need the following 3 preparations:
 2. Object storage is used to store data blocks
 3. JuiceFS Client
 
-Please refer to [Quick Start Guide](docs/en/getting-started/quick_start_guide.md) to start using JuiceFS immediately!
+Please refer to [Quick Start Guide](docs/en/getting-started/_quick_start_guide.md) to start using JuiceFS immediately!
```

Comment:
Thanks for your contribution! But the `_quick_start_guide.md` document will be delete later, the new "Quick Start Guide" is `for_local.md`. Could you update your PR?

Reply:
ok, i'll update later,thx